app/controllers/queue_items_controller.rb

```diff
@@ -39,8 +51,19 @@ def assign_new_order_position
   def no_duplicates?(video)
     if QueueItem.where(video_id: video.id, user_id: current_user.id).count == 0
       return true
-    else 
+    else
       false
     end
```

Comment:
This is not a good/idiomatic Ruby code:

```ruby
 if QueueItem.where(video_id: video.id, user_id: current_user.id).count == 0
  return true
else
  false
end
```

HOMEWORK: rework this to Ruby code, re-learn `return` keyword and `==` method.